html/pages/notifications.inc.php

```diff
@@ -232,6 +233,24 @@
     });
   });
 
+  $(document).on( "click", ".read-all-notif", function() {
+    $(this).attr("disabled", true);
+    $.ajax({
+      type: 'POST',
+      url: 'ajax_form.php',
+      data: {type: 'notifications', action: 'read-all-notif'},
+      dataType: "json",
+      success: function (data) {
+        if( data.status == "ok" ) {
+          window.location.reload()
```

Comment:
Should be able to hide all unread notifications and update the notification counts in jquery.